content/docs/05.developer-guide/13.cicd/06.bitbucket-pipes.md

```diff
@@ -0,0 +1,44 @@
+---
+title: BitBucket Pipes
+---
+
+## Setup a BitBucket Pipes
+
+Using a basic Kestra image and corresponding Kestra CLI, you can validate and deploy flow from BitBucket repositories through [BitBucket Pipes](https://support.atlassian.com/bitbucket-cloud/docs/configure-your-first-pipeline/).
+
+Here is a basic pipeline:
+
+
+```yaml
+
+image: kestra/kestra
+
+pipelines:
+  default:
+    - step:
+        name: 'Validate Kestra flows'
+        deployment: staging
+        script:
+          - /bin/sh /app/kestra flow validate flows/ --local
+
+    - step:
+        name: 'Deploy Kestra flows'
+        deployment: production
+        script:
+          - echo $SERVER
+          - echo $KESTRA_USER
+          - echo $KESTRA_PASSWORD
+          - /bin/sh /app/kestra flow namespace update dev flows/ --server=$SERVER --tenant=$TENANT --user=$KESTRA_USER:$KESTRA_PASSWORD
+```
+
+Variables such as `$SERVER`, `$KESTRA_USER`, `$KESTRA_PASSWORD` (and optionally `$TENANT` if you're using Enterprise Edition with tenants) are set in the BitBucket Variable configuration:
```

Comment:
Better to say: if you're using Kestra version supporting multi-tenancy

(Cloud and kestra professional support that too)